### PR TITLE
Update AD-FS-FAQ.md

### PR DESCRIPTION
**Pull request by rotraves-microsoft**
Correcting the command lines. Incorrectly parameters were being added on the previous version.

### CODE REVIEW

**Comment by PRMerger13** [comment]
@rotraves-microsoft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: WindowsServerDocs/identity/ad-fs/overview/AD-FS-FAQ.md [hunk 1]
```diff
@@ -276,16 +276,19 @@ On WAP servers you can still use Set-WebApplicationProxySslCertificate. On the A
...
 
 4.  Add the new cert
 
-    a. netsh http add sslcert hostnameport=fs.contoso.com:443 certhash=CERTTHUMBPRINT appid={5d89a20c-beab-4389-9447-324788eb944a} certstorename=MY sslctlstorename=AdfsTrustedDevices
+    a. netsh http add sslcert hostnameport=fs.contoso.com:443 certhash=THUMBPRINT appid="{5d89a20c-beab-4389-9447-324788eb944a}" certstorename=My verifyclientcertrevocation=Enable sslctlstorename=AdfsTrustedDevices
 
-    b. netsh http add sslcert hostnameport=localhost:443 certhash=CERTTHUMBPRINT appid={5d89a20c-beab-4389-9447-324788eb944a} certstorename=MY sslctlstorename=AdfsTrustedDevices
+    b. netsh http add sslcert hostnameport=localhost:443 certhash=THUMBPRINT appid="{5d89a20c-beab-4389-9447-324788eb944a}" certstorename=My verifyclientcertrevocation=Enable
+
+    c. netsh http add sslcert hostnameport=fs.contoso.com:49443 certhash=THUMBPRINT appid="{5d89a20c-beab-4389-9447-324788eb944a}" certstorename=My verifyclientcertrevocation=Enable clientcertnegotiation=Enable
```
Comment (illfated) [inline WindowsServerDocs/identity/ad-fs/overview/AD-FS-FAQ.md:291]: Due to GitHub reading line changes differently to us humans, I suggest removing this blank line to keep the same number of blank lines as the original paragraphs 4.a. - 4.c.
```suggestion
```
GitHub reads this as an extra blank line resulting from the addition of a blank line after `b. netsh http add sslcert [...]`

**Comment by PRMerger13** [comment]
@rotraves-microsoft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by illfated — APPROVED** [review]
Looks good to me.

**Comment by illfated** [comment]
#sign-off

**Comment by PRMerger12** [comment]
@illfated: I'm sorry - only the author of this article, @billmath, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by illfated** [comment]
Yes, I know. I mostly did it to get slightly more attention to this PR (as previously agreed on).

**Pull request merged**